app/services/planting_predictions.rb

```diff
@@ -3,13 +3,13 @@ def initialize(planting)
     @planting = planting
   end
 
-  def days_until_finished
+  def days_before_finished
```

Comment:
there was a mix of `until_finished` and '`before_maturity`
I've made them all use "before". Still needs renaming that column to `days_before_finished` to match the code and the UI.

Reply:
👍